src/ml.jl

```diff
@@ -0,0 +1,75 @@
+# Not correct for `map(pick(:foo), t)`
+Base.names(t::DTable) = fieldnames(eltype(t))
+Base.names(t::IndexedTable) = fieldnames(eltype(t))
+
+# Core schema types
+
+width(::Void) = 0
+numeric(::Void, x) = []
+
+struct Continuous{T}
+  μ::T
+  σ::T
+end
```

Comment:
@joshday can you think of something in the Stats/ML ecosystem we can reuse for this?

Reply:
`OnlineStats.Variance` calculates both mean and variance.  `StatsBase.zscore`?

Reply:
Sorry, that's not what you're asking... There's some overlap here with StatsModels and https://github.com/kleinschmidt/StreamModels.jl, but one is based on DataFrames and the other is a work in progress.  https://github.com/JuliaML/MLPreprocessing.jl takes a different approach but does something similar to`Continuous` here.